db/migrations/20190612135201_drop_title_from_movies.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
```

Comment:
In general we've tended to use `async/await` for DB migrations more recently. While for this one it's fine to keep it using promises for the sake of consistency in the repo, definitely take a look at some of the more recent migrations in `lob-api` and other repos when you write future migrations on Lob services.